src/api/prompt.ts

```diff
@@ -479,12 +483,15 @@ export async function keypress(context = Context.current): Promise<string> {
 
   return await new Promise<string>((resolve, reject) => {
     try {
+      let keys = "";
```

Comment:
nit: I try to only put throwing stuff in `try`, and to keep everything else out of it.

```suggestion
    let keys = "";

    try {
```